docs/README.md

```diff
@@ -6,25 +6,33 @@
 
 **Welcome to the _ZARP_ documentation pages!**
 
-_ZARP_ is a [Snakemake][snakemake] workflow ...
-
-**TODO:** Complete intro/summary section
-
-**NOTES TO SELF:**
-
-- Include references in `/includes/references.md`
-- Include abbreviations in `includes/abbreviations.md` (example: EAWNKWIMPR)
-- Include emojis (example: :rocket:)
-- Check out the [ZARP-cli docs pages](https://zavolab.github.io/zarp-cli/)
-for inspiration.
-- Check out [MKDocs](https://www.mkdocs.org/) and [Material for
-MkDocs](https://squidfunk.github.io/mkdocs-material/) docs for more info
+_ZARP_ is a generic RNA-Seq analysis workflow that allows users to process and analyze 
+Illumina short-read sequencing libraries with minimum effort. Better yet: With our
+companion [**ZARP-cli**](https://github.com/zavolanlab/zarp-cli) command line
+interface, you can start ZARP runs with the simplest and most intuitive commands.
+
+_RNA-seq analysis doesn't get simpler than that!_
+
+ZARP relies on publicly available bioinformatics tools and currently handles
+single or paired-end stranded bulk RNA-seq data. The workflow is developed in
+[Snakemake][snakemake], a widely used workflow management system in the
+bioinformatics community.
+
+ZARP will pre-process, align and quantify your single- or paired-end stranded
+bulk RNA-seq sequencing libraries with publicly available state-of-the-art
+bioinformatics tools. ZARP's browser-based rich reports and visualitations will
+give you meaningful initial insights in the quality and composition of your
+sequencing experiments - fast and simple. Whether you are an experimentalist
+struggling with large scale data analysis or an experienced bioinformatician,
+when there's RNA-seq data to analyze, just _zarp 'em_!
 
 ## How does it work?
 
-Briefly, when a _ZARP_ run is triggered, ...
+ZARP requires conda or mamba to install the basic dependencies. Each individual step of the workflow run either in its own Apptainer (Singularity) container or in its own Conda virtual environemnt.
 
-**TODO:** Complete section
+Once the installation is complete, you fill in a [config.yaml](https://github.com/zavolanlab/zarp/blob/dev/tests/input_files/config.yaml) file with parameters and a [samples.tsv](https://github.com/zavolanlab/zarp/blob/dev/tests/input_files/samples.tsv) file with sample specific information. You can easily trigger ZARP by making a call to snakemake with the appropriate parameters.
+
+The pipeline can be executed in different systems or HPC clusters. ZARP generates multiple output files that help you QC your data and proceed with downstream analyses. Apart from running the main ZARP workflow, you can also run a second pipeline that downloads data from SRA, and a third pipeline that populates a file with the samples and determines sample specific parameters.
```

Comment:
Maybe it would be good to be more precise:
- The SRA download workflow pulls _sequencing sample_ data, not just _any_ data.
- The HTSinfer workflow doesn't just populate _a file_ with samples, it tries to infer missing metadata from a sparse sample table (if I'm not mistaken).

Also, it would be good to spell out abbreviations like SRA and QC - people may not necessarily know, especially among those we want to address with ZARP.

Alternatively, MKDocs has a cool feature that allows you to define a glossary, and it will automatically create tooltips for defined terms/abbreviations in the glossary. Check it out, I'm pretty sure I'm using it in the ZARP-cli docs.

Reply:
Fixed. Thank you.